tsconfig.json

```diff
@@ -5,6 +5,7 @@
     "strict": false,
     "jsx": "preserve",
     "importHelpers": true,
+    "resolveJsonModule": true,
```

Comment:
This is needed to be able to load a JSON file (in FilingHistoryList2.spec.ts), ie,
`import { filings } from './filings.json'`